src/components/table-content-wrapper/table-content-wrapper.jsx

```diff
@@ -21,7 +21,8 @@ class TableContentWrapper extends Component {
   render() {
     const {
       classes,
-      captionHidden,
```

Comment:
are we changing/breaking the api here?

Reply:
I added this yesterday night

Reply:
Can we in some way merge together my suggestion below and keeping `captionHidden`?

Reply:
But why? No one has used this, it's only me working on this